api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/OpenEmpiPatientQuery.java

```diff
@@ -0,0 +1,191 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.Date;
+import java.util.List;
+
+@XmlRootElement(name = "person")
+public class OpenEmpiPatientQuery {
```

Comment:
Change this class name to something like OpenEmpiPatientResult. (This is the result, not the query, right?)